BrownianMotion/StochasticIntegral/Locally.lean

```diff
@@ -432,31 +432,62 @@ lemma locally_of_ae {p : (ι → E) → Prop} (hpX : ∀ᵐ ω ∂P, p (X · ω)
     · simp [LocalizingSequence_of_prop, if_neg hω]
     · simp [LocalizingSequence_of_prop, if_neg hω]
 
-section NormedSpace
+section TopologicalSpace
 
-variable [NormedSpace ℝ E] [CompleteSpace E]
+variable [TopologicalSpace E]
 
+omit [HasUsualConditions 𝓕 P] in
```

Comment:
Can you move the lemma groupings so that we don't need `omit [HasUsualConditions 𝓕 P] in`